attributioncode/validator.go

```diff
@@ -201,3 +215,14 @@ func checkMAC(key, msg, msgMAC []byte) error {
 	}
 	return nil
 }
+
+func fromRTAMO(content string) bool {
+	matched, err := regexp.MatchString(`^rta:`, content)
+	if err != nil {
+		return false
+	}
+	if matched {
+		return true
+	}
+	return false
```

Comment:
I haven't written any Go recently but could you return `matched` directly?

```suggestion
	return matched
```

Reply:
ah, indeed you can. Thanks! :slightly_smiling_face: